### PR TITLE
[TASK] Revise chapter about XML sitemaps

### PR DESCRIPTION
**Pull request by brotkrueml**
Releases: main, 11.5

### CODE REVIEW

File: Documentation/ApiOverview/Seo/XmlSitemap.rst [hunk 1]
```diff
@@ -1,263 +1,299 @@
...
-
-It is possible to generate XML sitemaps for SEO purposes without using 3rd-party plugins.
-When enabled, this new feature will create a sitemapindex with one or more sitemaps in it.
-Out-of-the-box it will have one sitemap containing all the pages of the current site and
-language. Per site and per language you have the possibility to render a different sitemap.
+It is possible to generate XML sitemaps for SEO purposes without using 3rd party
+plugins. When this feature is enabled, a sitemap index file is created with one
+or more sitemaps in it. By default, there will be one sitemap that contains all
+pages of the current site and language. Per each site and language, you can
+render a different sitemap.
```
Comment (linawolf) [inline Documentation/ApiOverview/Seo/XmlSitemap.rst:13]:
```suggestion
pages of the current site and language.  You can render different sitemaps
for each site and language.
```

File: Documentation/ApiOverview/Seo/XmlSitemap.rst [hunk 2]
```diff
@@ -1,263 +1,299 @@
...
+   :depth: 1
+   :local:
+
 Installation
 ============
 
-The XML sitemap is disabled by default. You can easily enable it by installing the system
-extension "seo" and including the static TypoScript template XML Sitemap (seo). It is also
-mandatory to have a site configuration for your rootpage(s).
+The XML sitemap is not available by default. You can easily activate it by
```
Comment (linawolf) [inline Documentation/ApiOverview/Seo/XmlSitemap.rst:27]: What do you consider default? In the TYPO3/DEFAULT package of the composer helper seo is already contained and activated: https://get.typo3.org/misc/composer/helper

Comment (brotkrueml) [reply Documentation/ApiOverview/Seo/XmlSitemap.rst:27]: I considered typo3/minimal: https://packagist.org/packages/typo3/minimal

Comment (brotkrueml) [reply Documentation/ApiOverview/Seo/XmlSitemap.rst:27]: Maybe: "The seo system extension provides XML sitemaps. If it is not available on your installation, you can easily ..."

Comment (linawolf) [reply Documentation/ApiOverview/Seo/XmlSitemap.rst]: Better, but avoid easily, this frustrates people who will have trouble installing

File: Documentation/ApiOverview/Seo/XmlSitemap.rst [hunk 3]
```diff
@@ -1,263 +1,299 @@
...
+
 Installation
 ============
 
-The XML sitemap is disabled by default. You can easily enable it by installing the system
-extension "seo" and including the static TypoScript template XML Sitemap (seo). It is also
-mandatory to have a site configuration for your rootpage(s).
+XML sitemaps are part of the "seo" system extension. If the extension is not
+available in your installation, it can be easily activated by installing it and
+including the static TypoScript template `XML Sitemap (seo)`.
```
Comment (linawolf) [inline Documentation/ApiOverview/Seo/XmlSitemap.rst:29]:
```suggestion
available in your installation, require it as described here: :ref:`Installation, EXT:seo <ext_seo:installation>`
Then include the static TypoScript template `XML Sitemap (seo)`.
```

**Review by linawolf — APPROVED** [review]

**Pull request merged**